.for_maintainers/release/RELEASE_NOTES_TEMPLATE.md

```diff
@@ -0,0 +1,167 @@
+```
+
+To get a new copy of the Wallaroo repository, run the following commands:
+
+```bash
+cd ~/wallaroo-tutorial/
+curl -L -o wallaroo-x.x.x.tar.gz 'https://wallaroo-labs.bintray.com/wallaroolabs-ftp/wallaroo/x.x.x/wallaroo-x.x.x.tar.gz'
+mkdir wallaroo-x.x.x
+tar -C wallaroo-x.x.x --strip-components=1 -xzf wallaroo-x.x.x.tar.gz
+rm wallaroo-x.x.x.tar.gz
+cd wallaroo-x.x.x
+```
+
+You can then run the following commands to build the necessary tools to continue developing using Wallaroo x.x.x:
+
+```bash
+cd ~/wallaroo-tutorial/wallaroo
+make build-machida build-giles-all build-utils-cluster_shutdown
+```
+
+<a name="upgrading-wallaroo-in-docker"></a>
+### Upgrading the Wallaroo Docker image
+
+To upgrade the Wallaroo Docker image, run the following command to get the latest image.  If you don't allow a non-root user to run Docker commands, you'll need to add `sudo` to the front of the command.
+
+```bash
+docker pull wallaroo-labs-docker-wallaroolabs.bintray.io/release/wallaroo:x.x.x
+```
+
+#### Upgrading Wallaroo Source Code
+
+If you mounted the Wallaroo source code to your local machine using the directory recommended in setup, in  `/tmp/wallaroo-docker` (UNIX & MacOS users) or `c:/wallaroo-docker` (Windows users), then you will need to move the existing directory in order to get the latest source code.  The latest Wallaroo source code will be copied to this directory automatically when a new container is started with the latest Docker image.
+
+##### UNIX & MacOS Users
+For UNIX users, you can move the directory with the following command:
+
+```bash
+mv /tmp/wallaroo-docker/wallaroo-src/ /tmp/wallaroo-docker/wallaroo-x.x.x-src/
+```
+
+##### Windows Users
+For Windows users, you can move the directory with the following command:
+```bash
+move c:/wallaroo-docker/wallaroo-src/ c:/wallaroo-docker/wallaroo-x.x.x-src
+```
+
+
+Once done moving, you can re-create the `wallaroo-src` directory with the following command:
+
+```bash
+mkdir c:\wallaroo-docker\wallaroo-src
+```
+
+<a name="upgrading-wallaroo-in-vagrant"></a>
+### Upgrading Wallaroo in Vagrant
+
+To upgrade your Wallaroo installation in Vagrant, you’ll want to start by moving your current Vagrant directory to a new location. Assuming that you’ve installed it according to our setup documentation, you’ll run:
+
+```bash
+cd ~/
+mv wallaroo-tutorial/wallaroo/vagrant wallaroo-tutorial/wallaroo/vagrant-x.x.x
+cd ~/wallaroo-tutorial/wallaroo/
+git fetch origin
+git checkout -f x.x.x
+```
+
+Finally, to provision your new Vagrant box, run the following commands:
+
+```bash
+cd ~/wallaroo-tutorial/wallaroo/vagrant
+vagrant up
+```
+
+If you have modified your old Vagrant VM in any way that you intend to persist, you’ll need to do that now.  For example, copy any edited or new files from the old Vagrant VM to the new one. When you’ve completed that, it’s a good idea to clean up your old Vagrant box, by running:
+
+```bash
+cd ~/wallaroo-tutorial/wallaroo/vagrant-x.x.x
```

Comment:
This should refer to the `wallaroo-x.x.x` directory instead of the `wallaroo` directory.